modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2804,11 +2806,20 @@ export class Wallet implements IWallet {
       throw new Error('prv required to sign transactions with TSS');
     }
     try {
+      const hashAlgorithm = this.coinSpecific()?.hashAlgorithm;
+      let hash: Hash;
+      if (hashAlgorithm && hashAlgorithm.startsWith('sha')) {
+        hash = createHash(hashAlgorithm);
+      } else {
+        hash = createKeccakHash('keccak256');
+      }
+
```

Comment:
```suggestion
      const hashAlgorithm = this.coinSpecific()?.hashAlgorithm || 'keccak256';
      const hash = createHash(hashAlgorithm);
```

Reply:
I think this might be a bit better approach to do this

Reply:
Had a conversation with @dpkjnr , Seems Keccack is not readily available with node crypto and have to use an external dependency to support the same